### PR TITLE
Fix links reported as broken

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
https://github.com/pypa/packaging.python.org/pull/975#issuecomment-912519338

Note: many sections on the docs of Setuptools don't have intersphinx references.

### CODE REVIEW

File: source/overview.rst [hunk 1]
```diff
@@ -206,7 +206,7 @@ respective packaging guides.
 * `Google App Engine <https://cloud.google.com/appengine/docs/python/>`_
 * `PythonAnywhere <https://www.pythonanywhere.com/>`_
 * `OpenShift <https://blog.openshift.com/getting-started-python/>`_
-* "Serverless" frameworks like `Zappa <https://www.zappa.io/>`_
+* "Serverless" frameworks like :gh:`Zappa <zappa/Zappa>`
```
[the remark this answers is not in the document]
Comment (meowmeowmeowcat) [reply source/overview.rst:209]: I think there is no website anymore. Their Github homepage points to here: https://github.com/zappa/Zappa

File: source/conf.py [hunk 2]
```diff
@@ -394,6 +399,9 @@
     'pip': ('https://pip.pypa.io/en/latest/', None),
     'setuptools': ('https://setuptools.readthedocs.io/en/latest/', None),
     'twine': ('https://twine.readthedocs.io/en/stable/', None),
+    'tox': ('https://tox.readthedocs.io/en/latest/', None),
```
[the remark this answers is not in the document]
Comment (meowmeowmeowcat) [reply source/conf.py:402]: Oh, I just realize that they are arranged in alphabetical order.

File: source/guides/supporting-multiple-python-versions.rst [hunk 3]
```diff
@@ -88,7 +88,7 @@ with a single-source strategy, there are a number of options.
 Tools for single-source Python packages
 ----------------------------------------
 
-`six <http://pythonhosted.org/six/>`_ is a tool developed by Benjamin Peterson
+`six <https://pypi.org/project/six/>`_ is a tool developed by Benjamin Peterson
```
[the remark this answers is not in the document]
Comment (meowmeowmeowcat) [reply source/guides/supporting-multiple-python-versions.rst:91]: In Line 96, project `modernize` points to `https://pypi.org/project/modernize`. Would you prefer making them unified or just use RTD link here?

[the remark this answers is not in the document]
Comment (webknjaz) [reply source/guides/supporting-multiple-python-versions.rst:91]: Go for the same style as others.

**Comment by meowmeowmeowcat** [comment]
Sorry that I made a careless mistake when trying to use `git rebase`. I've opened #982 for a new PR of this.

**Pull request closed without merging**